src/ucs/type/spinlock.h

```diff
@@ -73,11 +73,18 @@ void ucs_spinlock_destroy(ucs_spinlock_t *lock);
 void ucs_recursive_spinlock_destroy(ucs_recursive_spinlock_t *lock);
 
 static inline int
-ucs_recursive_spin_is_owner(ucs_recursive_spinlock_t *lock, pthread_t self)
+ucs_recursive_spin_is_owner(const ucs_recursive_spinlock_t *lock,
```

Comment:
seems no need to break line: unrelated and does not exceed 80 syms

Reply:
it doesn't fit the 80 symbols per line requirement (it is `81`)